_posts/2024-02-27-the-definitive-guide-2-jp.md

```diff
@@ -0,0 +1,43 @@
+---
+layout: post
+title:  "Japanese edition of Trino: The Definitive Guide"
+author: Yuya Ebihara
+image: /assets/ttdg2-jp-cover.jpg
+excerpt_separator: <!--more-->
+show_pagenav: false
+---
+
+Do you know where the name 'Trino' comes from? It's actually a shortened form of 'neutrino'.
+These fast and lightweight subatomic particles have recently made their way to Japan.
+The reservation for the Japanese edition of 'Trino: The Definitive Guide' have finally begun.
+
+<!--more-->
+
+Today, we are happy to announce that a Japanese translation of the book [Trino:
+The Definitive Guide]({{ site.url }}/trino-the-definitive-guide.html) will be
+published for the communities all across Japan and far beyond and hopefully a
+lowers the barrier to Trino for native speakers. We invite you all to get your
+own copy:
+
+<div class="card-deck spacer-30">
+    <a class="btn btn-pink" href="https://www.hanmoto.com/bd/isbn/9784798071671">
+        分散SQLクエリエンジンTrino徹底ガイド 秀和システム
+    </a>
+</div>
+<div class="spacer-30"></div>
+
+Our thanks goes out Masanori Nishida and his teams at Shuwa System and Kai Sasaki, Akira Ajisaka, Kaname Nishizuka and Miki Takata for making this happen.
```

Comment:
Do you have links for all these people? website, linkedin, or github profile or whatever?

Reply:
Great